pkg/config/agent/config.go

```diff
@@ -61,6 +61,11 @@ type AgentConfig struct {
 	// the external network needs not be SNAT'd. In the networkPolicyOnly mode, antrea-agent never
 	// performs SNAT and this option will be ignored; for other modes it must be set to false.
 	NoSNAT bool `yaml:"noSNAT,omitempty"`
+	// Fully randomize source port mapping in SNAT rules enforced by Nodes for egress traffic
+	// from local Pods to the external network. This has no impact on the SNAT rules for the
+	// Egress feature. This has no impact on the SNAT rules for the Egress feature. Default is
+	// false.
+	SNATFullyRandomPorts bool `yaml:"snatFullyRandomPorts"`
```

Comment:
@tnqn I opted for `bool` here instead of `*bool`. If we want to change the default value to `true` in the future, I believe we can switch out for a `*bool` at that point.

Reply:
It works for me.